src/signals/incident/containers/KtoContainer/components/KtoForm/index.js

```diff
@@ -147,6 +150,7 @@ const KtoForm = ({
           <StyledLabel as="legend" ref={firstLabelRef}>
             Waarom bent u {!isSatisfied ? 'on' : ''}tevreden?
           </StyledLabel>
+          <SubTitle> U kunt meer keuzes maken.</SubTitle>
```

Comment:
It is unnecessary to make a new component. You can use the HelpText styled component here.

Reply:
Done. Changed the component to HelpText.